### PR TITLE
Added Rest communication and modified the access token

### PR DESCRIPTION
**Pull request by Sri-Harsha06**

### CODE REVIEW

File: generator-jhipster-gomicro/generators/server/generator.mjs [hunk 1]
```diff
@@ -121,6 +123,28 @@ export default class extends ServerGenerator {
...
+            } else if (options.framework === 'rabbitmq') {
+              if (options.server)
+                rabbitmqServer = options.server;
+              if (options.client)
+                rabbitmqClient = options.client;
+            }
+          }
+        }
+        
+        console.log("%%%%%%%%%%%%%%",restServer,restClient)
```
Comment (craxkumar) [inline generator-jhipster-gomicro/generators/server/generator.mjs]: Please remove this log or have some meaning full log,
having a meaning full log would be a good option

**Review by craxkumar — DISMISSED** [review]
Review OK!, please make this minor change

**Review by craxkumar — APPROVED** [review]
Review OK!

**Review by rohithteja-cmi — APPROVED** [review]

**Review by craxkumar — APPROVED** [review]
Review OK!

**Pull request merged**